### PR TITLE
added Firefox OS

### PR DESCRIPTION
**Pull request by marti1125**
please check my PR

### CODE REVIEW

File: medic/medic-run.js [hunk 1]
```diff
@@ -333,6 +333,25 @@ function main() {
...
+        // run the code
+        // NOTE:
+        //      this is ASYNCHRONOUS
+        util.medicLog("running:");
+        util.medicLog("    " + runServe);
+        shelljs.exec(runServe, {silent: false, async: true}, function (returnCode, output) {
+            if (returnCode != 0 || CORDOVA_ERROR_PATTERN.test(output)) {
+                util.fatal("run failed");
+            } else {
+                window.open("http://localhost:8000/firefoxos/www/");
```
Comment (dblotsky) [inline medic/medic-run.js]: This script is run from node, which does not have the `window` object. To open the app you will need to launch a real browser via command line.

**Comment by marti1125** [comment]
I added this lib https://www.npmjs.com/package/open

**Comment by purplecabbage** [comment]
you will need to add the dependency to package.json also.

**Comment by marti1125** [comment]
any update about it?

**Comment by marti1125** [comment]
this PR is generating conflict with some files....

**Pull request closed without merging**